src/common/constants.ts

```diff
@@ -1,7 +1,18 @@
 export const ANDROID = 'ANDROID';
 export const DAYS_IN_WEEK = 7;
 export const DEFAULT_LIMIT = 25.0;
-export const FILTERED_WORDS = ['covid-19', 'covid', 'coronavirus', 'masks', 'mask','pandemic','test','testing','tests'];
+export const DEFAULT_OFFSET = 1;
```

Comment:
Why is the default offset 1? Would that skip the first article each query?

Reply:
Updated should be good to go now